messages/run-command.md

```diff
@@ -1,39 +1,90 @@
+
+You can combine different criteria using colons to further narrow the list of rules to run; the colon works as an intersection.  For example, "--rule-selector eslint:Security" lists rules associated only with the Eslint engine that have the Security tag.  To add multiple rule selectors together (a union), specify the --rule-selector flag multiple times, such as "--rule-selector eslint:default --rule-selector retire-js:default:3".
+
+If you've created a custom configuration file that overrides the default rule and engine properties, you can specify it with the --config-file flag.
+
+By default, the command outputs the results in the terminal. Use the --output-file flag to print the results to a file; the format of the results depends on the extension you provide. For example, "--output-file results.csv" creates a comma-separated values file. You can specify one of these extensions:
+
+- .csv
+- .html or .htm
+- .json
+- .junit or .junit.xml
+- .sarif or .sarif.json
+- .xml 
 
 # command.examples
 
-*DRAFT*: Command examples
+- Analyze the files in the current folder using the recommended rules; use concise table output:
+
+    <%= config.bin %> <%= command.id %>
+
+- Analyze the files in the current folder using all available rules, but show details of the violations:
+
+    <%= config.bin %> <%= command.id %> --rule-selector all --view detail
+
+- Analyze only two files, and fail if the run encounters a violation of with severity level of "high" or higher:
+
+    <%= config.bin %> <%= command.id %> --workspace ./my-workspace/init.ts --workspace ./my-workspace/dosomething.ts --severity-threshold high
+
+- Specify a custom configuration file and output the results to a file in CSV format, with severity threshold of moderate (3) :
+
+    <%= config.bin %> <%= command.id %> --config-file ./code-analyzer.yml --output-file results.csv --severity-threshold 3
+
+- Analyze the files in the specified workspace, but run only ESLint rules that have a moderate severity (3) and the default RetireJS rules with any severity:
+
+    <%= config.bin %> <%= command.id %> --workspace ./my-workspace --rule-selector eslint:3 --rule-selector retire-js:default
+
+- Analyze the files in the current folder, but run only the "getter-return" ESLint rule and the "no-inner-declarations" rule in any engine:
+
+    <%= config.bin %> <%= command.id %> --rule-selector eslint:getter-return --rule-selector no-inner-declarations
+
+- Specify method starting points of the analysis:
+
+    <%= config.bin %> <%= command.id %> --path-start ./classes/Utils.cls#init --path-start ./classes/Helpers.cls#method1;method2
 
 # flags.workspace.summary
 
-*DRAFT*: Flag summary
+Set of files to be included in the code analysis. You can specify specific files or directories, and use glob patterns (wildcards). 
```

Comment:
As mentioned, we don't currently support globs, though we intend to in the long-term.

Reply:
I'm going to leave some info about globs, as we discussed.